### PR TITLE
chore: replacing links to tokens lists to updated ones

### PR DESCRIPTION
**Pull request by Shani-roz**
### Description of the Changes

I updated the links to the tokens lists on mainnet and sepolia to the json files that contain all the tokens.

### PR Preview URL

https://starknet-io.github.io/starknet-docs/pr-1381/starkgate/overview/

### Check List

- [x] Changes have been done against main branch, and PR does not conflict
- [x] PR title follows the convention: `<docs/feat/fix/chore>(optional scope): <description>`, e.g: `fix: minor typos in code`

### CODE REVIEW

**Comment by LandauRaz** [comment]
Hey @Shani-roz, thank you very much for the contribution 🙏
To expedite the review of this PR, can you please:
1. Add the specific URL(s) of the content that this PR addresses under "PR Preview URL"
2. Edit the PR title according to the convention specified under "Check List"
Thanks!

File: components/Starknet/modules/starkgate/pages/overview.adoc [hunk 1]
```diff
@@ -14,15 +14,15 @@ include::partial$snippet_backwards_compatibility_note.adoc[]
...
+L1 and L2 addresses for StarkGate bridges and supported tokens are listed in the JSON files shown in the table xref:#table_StarkGate_token_addresses[].
 
 [#table_StarkGate_token_addresses]
 .StarkGate bridged tokens and their addresses
 |===
 | Network | StarkGate bridged tokens JSON file
 
-| Mainnet | link:https://github.com/starknet-io/starknet-addresses/blob/master/bridged_tokens/mainnet.json[mainnet.json]
-| Sepolia testnet | link:https://github.com/starknet-io/starknet-addresses/blob/master/bridged_tokens/sepolia.json[sepolia.json]
+| Mainnet | link:https://beta.starkgate.starknet.io/static/tokens.json[mainnet.json]
```
Comment (LandauRaz) [inline components/Starknet/modules/starkgate/pages/overview.adoc:24]: These files are no longer called "mainnet.json" and "sepolia.json" anymore, so please remove their names (i.e., the square bracket parts)
Please also change "Sepolia testnet" to "Sepolia".

File: components/Starknet/modules/starkgate/pages/overview.adoc [hunk 2]
```diff
@@ -14,15 +14,15 @@ include::partial$snippet_backwards_compatibility_note.adoc[]
 [#starkgate_addresses]
 == StarkGate addresses
 
-L1 and L2 addresses for StarkGate bridges and supported tokens are listed in the JSON files in the Starknet GitHub repository shown in the table xref:#table_StarkGate_token_addresses[].
+L1 and L2 addresses for StarkGate bridges and supported tokens are listed in the JSON files shown in the table xref:#table_StarkGate_token_addresses[].
```
Comment (LandauRaz) [inline components/Starknet/modules/starkgate/pages/overview.adoc:17]: Please also remove the "in the Starknet GitHub repository" part in the second paragraph of "Supported tokens in StarkGate" (currently line 33) as well.

**Review by LandauRaz — CHANGES_REQUESTED** [review]

**Comment by LandauRaz** [comment]
@Shani-roz, I left a couple of small comments and will merge the PR as soon as you address them.
Also, thanks for adding the URL under "PR Preview URL" and adding a title according to the convention specified under "Check List", but please note how I edited both for future PRs.

**Review by LandauRaz — CHANGES_REQUESTED** [review]

File: components/Starknet/modules/starkgate/pages/overview.adoc [hunk 3]
```diff
@@ -14,23 +14,23 @@ include::partial$snippet_backwards_compatibility_note.adoc[]
...
 
 [#table_StarkGate_token_addresses]
 .StarkGate bridged tokens and their addresses
 |===
-| Network | StarkGate bridged tokens JSON file
+| StarkGate bridged tokens JSON files
```
[the remark this answers is not in the document]
Comment (Shani-roz) [reply components/Starknet/modules/starkgate/pages/overview.adoc:22]: I know that's not what you meant but I thought that just removing the links' names and show the URLs also wouldn't look so good. wdyt?

[the remark this answers is not in the document]
Comment (LandauRaz) [reply components/Starknet/modules/starkgate/pages/overview.adoc:22]: 1. I think removing the links' names and showing the URLs is still better. You can shorten them a bit, e.g. `https://beta.starkgate.starknet.io/static/tokens.json` -> `beta.starkgate.starknet.io/tokens.json`
2. You also can just call both of them `tokens.json`
Up to you!

**Review by LandauRaz — APPROVED** [review]

**Pull request merged**